model/database_functions.ts

```diff
@@ -0,0 +1,68 @@
+import { database } from "./createDatabase";
+import { Journey, Trip, Location } from "./model";
+
+const createJourney = async (title: string): Promise<Journey> => {
+  return await database.write(async () => {
+    const journey = await database
+      .get<Journey>("journeys")
+      .create((journey) => {
+        journey.title = title;
+      });
+    return journey;
+  });
```

Comment:
```suggestion
  return void database.write(async () => {
    return database
      .get<Journey>("journeys")
      .create((journey) => {
        journey.title = title;
      });
  });
```

Wäre es nicht besser journey direkt zu returnen anstatt es in eine Variable zu schreiben? Was zurückgegeben wird sieht man ja am Type. Außerdem ist return await in diesem Kontext dasselbe wie return.